server/core/server.go

```diff
@@ -74,18 +76,24 @@ type AccountServer struct {
 
 // NewAccountServer creates a new account server with a default logger
 func NewAccountServer() *AccountServer {
-	return &AccountServer{
-		logger: logging.NewNATSLogger(logging.Config{
-			Colors: true,
-			Time:   true,
-			Debug:  true,
-			Trace:  true,
-		}),
+	ac := &AccountServer{
+		logger: NewNilLogger(),
```

Comment:
That's all good to have a NilLogger for the tests, however having this here does not help. I have ran the tests locally and still can see logging. The reason is because the logger is set based on a given config (different than this one) in server.Start().